lgsm/config-default/config-lgsm/armarserver/_default.cfg

```diff
@@ -0,0 +1,183 @@
+##################################
+######## Default Settings ########
+##################################
+# DO NOT EDIT, ANY CHANGES WILL BE OVERWRITTEN!
+# Copy settings from here and use them in either:
+# common.cfg - applies settings to every instance.
+# [instance].cfg - applies settings to a specific instance.
+
+#### Game Server Settings ####
+
+# Queryport cannot be changed according to the wiki
+# https://community.bistudio.com/wiki/Arma_Reforger:Server_Hosting
+queryport="17777"
```

Comment:
Currently there is no steam query port. https://feedback.bistudio.com/T165100

However, queryport could be set to 1376 and moved to fn_info_game_armar to have query monitor support.

Reply:
What does this port do ?
I cannot find any info about this.

Reply:
I honestly have no idea, this tcp port is not documented anywhere. However, if you run "netstat -tlunp" you will see it is there. I did verify that if the server hangs, this port will go down. I think at the moment this is the only workaround to use server query monitoring.

Reply:
I try to get some information about query port on Arma discord, but nobody knows. So I search for undocumented config variables and I found something.
I found that https://community.bistudio.com/wiki/Arma_Reforger:Startup_Parameters#Hosting have options to change a2s ip and port (a2s is steam query protocol). But this options is not necessary.
Most important is to enable steam query is needed `a2sQueryEnabled` set to `true` in config. Also, query port can be change by `steamQueryPort`, but if not set they use 17777. I PR this to config repo. Monitor looks like work right now.

Reply:
Nice find @Marcin648!

I see there are some issues with this a2s query reporting, but still, we should be fine to use it for query monitoring.

Reply:
Merged the config  update.

Reply:
Changed this now with the new coinfig update.